.golangci.yaml

```
@@ -44,6 +45,286 @@ issues:
+        alias: apiextensions
+      - pkg: github.com/gardener/gardener/pkg/apis/core
+        alias: gardencore
+      - pkg: github.com/gardener/gardener/pkg/apis/core/v1beta1
+        alias: gardencorev1beta1
+      - pkg: github.com/gardener/gardener/pkg/apis/core/v1alpha1
+        alias: gardencorev1alpha1
+      - pkg: github.com/gardener/gardener/pkg/apis/extensions/v1alpha1"
+        alias: extensionsv1alpha1
+      - pkg: github.com/gardener/gardener/pkg/apis/operations/v1alpha1
+        alias: operationsv1alpha1
+      - pkg: github.com/gardener/gardener/pkg/apis/resources/v1alpha1
+        alias: resourcesv1alpha1
+      - pkg: github.com/gardener/gardener/pkg/apis/seedmanagement/v1alpha1
+        alias: seedmanagementv1alpha1
+      - pkg: github.com/gardener/gardener/pkg/apis/settings/v1alpha1
+        alias: settingsv1alpha1
+      - pkg: github.com/gardener/gardener/pkg/apis/core/v1alpha1/helper
+        alias: gardencorev1alpha1helper
+      - pkg: github.com/gardener/gardener/pkg/apis/core/v1beta1/constants
+        alias: v1beta1constants
+      - pkg: github.com/gardener/gardener/pkg/apis/core/v1beta1/helper
+        alias: gardencorev1beta1helper
+      - pkg: github.com/gardener/gardener/pkg/apis/core/install
+        alias: gardencoreinstall
+      - pkg: github.com/gardener/gardener/pkg/apis/seedmanagement/install
+        alias: seedmanagementinstall
+      - pkg: github.com/gardener/gardener/pkg/apis/settings/install
+        alias: settingsinstall
+      - pkg: github.com/gardener/gardener/pkg/admissioncontroller/apis/config
+        alias: admissioncontrollerconfig
+      - pkg: github.com/gardener/gardener/pkg/admissioncontroller/apis/config/v1alpha1
+        alias: admissioncontrollerconfigv1alpha1
+      - pkg: github.com/gardener/gardener/pkg/admissioncontroller/apis/config/validation
+        alias: admissioncontrollerconfigvalidation
+      - pkg: github.com/gardener/gardener/pkg/apiserver/admission/initializer
+        alias: admissioninitializer
+      - pkg: github.com/gardener/gardener/pkg/chartrenderer
+        alias: chartrenderer
+      - pkg: github.com/gardener/gardener/pkg/client/core/clientset/internalversion
+        alias: gardencoreclientset
+      - pkg: github.com/gardener/gardener/pkg/client/core/clientset/versioned
+        alias: gardenversionedcoreclientset
+      - pkg: github.com/gardener/gardener/pkg/client/core/informers/internalversion
+        alias: gardencoreinformers
+      - pkg: github.com/gardener/gardener/pkg/client/core/informers/externalversions
+        alias: gardenexternalcoreinformers
+      - pkg: github.com/gardener/gardener/pkg/client/kubernetes/clientmap/builder
+        alias: clientmapbuilder
+      - pkg: github.com/gardener/gardener/pkg/client/seedmanagement/clientset/versioned
+        alias: seedmanagementclientset
+      - pkg: github.com/gardener/gardener/pkg/client/seedmanagement/informers/externalversions
+        alias: seedmanagementinformer
```

To be consistent with the above convention `gardencoreinformers`
```suggestion
        alias: seedmanagementinformers
```